apiserver/admin_test.go

```diff
@@ -1003,10 +1032,23 @@ func (s *loginSuite) TestAuditLoggingFailurePreventsLogin(c *gc.C) {
 	request := &params.LoginRequest{
 		AuthTag:     user.Tag().String(),
 		Credentials: password,
-		CLIArgs:     "hey you guys",
+
```

Comment:
Probably, don't need a newline?

Reply:
Oops, due to some inexpert keyboard mashing.